### PR TITLE
chore(operator): add openebs operator 2.8.0

### PR DESCRIPTION
**Pull request by akhilerm**
- cstor operator 2.8.0
- jiva operator 2.8.0
- lvm operator 0.4.0
- ndm operator 1.4.0
- openebs-operator 2.8.0
- zfs operator 1.6.0

Signed-off-by: Akhil Mohan <akhil.mohan@mayadata.io>

### CODE REVIEW

**Comment by pawanpraka1** [comment]
@akhilerm can you update the operator yamls for zfs and lvm from the repo as we have started using 0.4.0 yaml generator.

**Comment by akhilerm** [comment]
Done @pawanpraka1

**Review by pawanpraka1 — COMMENTED** [review]
LVM-LocalPV RC1 and ZFS-LocalPV RC1 and RC2 yamls are fine.

File: 2.8.0/jiva-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,3781 @@
...
+            limits:
+              cpu: 100m
+              memory: 30Mi
+            requests:
+              cpu: 100m
+              memory: 20Mi
+      terminationGracePeriodSeconds: 10
+---
+
+apiVersion: storage.k8s.io/v1beta1
```
Comment (shubham14bajpai) [inline 2.8.0/jiva-operator.yaml:3333]: Can we update this to v1

File: 2.8.0/openebs-operator-lite.yaml [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+# Create Maya Service Account
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: openebs-maya-operator
+  namespace: openebs
+---
+# Define Role that allows operations on K8s pods/deployments
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
```
Comment (shubham14bajpai) [inline 2.8.0/openebs-operator-lite.yaml:19]: Same comment

File: 2.8.0/openebs-operator-lite.yaml [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+- apiGroups: ["openebs.io"]
+  resources: [ "*"]
+  verbs: ["*"]
+- nonResourceURLs: ["/metrics"]
+  verbs: ["get"]
+---
+# Bind the Service Account with the Role Privileges.
+# TODO: Check if default account also needs to be there
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
```
Comment (shubham14bajpai) [inline 2.8.0/openebs-operator-lite.yaml:53]: same comment

**Review by pawanpraka1 — APPROVED** [review]
zfs and lvm localpv operator yamls are fine.

File: 2.8.0/jiva-operator.yaml (hunk 1 above)
Comment (akhilerm) [reply 2.8.0/jiva-operator.yaml]: Done.

File: 2.8.0/openebs-operator-lite.yaml (hunk 2 above)
Comment (akhilerm) [reply 2.8.0/openebs-operator-lite.yaml]: Done.

File: 2.8.0/openebs-operator-lite.yaml (hunk 3 above)
Comment (akhilerm) [reply 2.8.0/openebs-operator-lite.yaml]: Done.

**Review by shubham14bajpai — APPROVED** [review]
LGTM

**Review by w3aman — APPROVED** [review]
zfs and lvm operator files /lgtm

**Review by kmova — APPROVED** [review]

**Pull request merged**